### PR TITLE
Animated stopwatch created

### PR DESCRIPTION
**Pull request by roman-zherebetskyi**
- [DEMO LINK](https://roman-zherebetskyi.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="page">
+    <div class="watch">
+      <div class="center-point"></div>
```
Comment (VikaMartin22) [inline src/index.html:13]: You should use BEM naming, like: watch > watch__center-point

File: src/styles/utils/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@mixin watch-size($page-height) {
```
Comment (VikaMartin22) [inline src/styles/utils/_mixins.scss:1]: It would be better to use abstract name for this mixin
'Cause you use it not only for watch container.

**Review by VikaMartin22 — COMMENTED** [review]
Good work, small naming corrections

**Review by VikaMartin22 — APPROVED** [review]

File: src/styles/utils/_mixins.scss (hunk 2 above)
Comment (roman-zherebetskyi) [reply src/styles/utils/_mixins.scss:1]: I'm agree with you

**Review by MaksimusCRS — APPROVED** [review]
GJ

**Pull request closed without merging**